components/dashboard/src/teams/Members.tsx

```diff
@@ -70,6 +75,33 @@ export default function() {
         }
     }
 
+    const removeTeamMember = async (userId: string) => {
+        await getGitpodService().server.removeTeamMember(team!.id, userId);
```

Comment:
todo: add a `try-catch` to handle the auth error

Reply:
Solved by removing the option when you're not an owner.